helm-charts/falcon-image-analyzer/values.schema.json

```diff
@@ -133,7 +133,7 @@
         "crowdstrikeConfig": {
             "type": "object",
             "required": [
-                "clusterName"
+                "cid"
```

Comment:
`cid` should not be required here in the schema. This will break the unified Helm chart's usage of `global.falcon.cid`. Instead of adding `cid` as required here. I've added validation helpers in the other Helm charts to account for either an inherited global CID or the component specific CID.

ref: https://github.com/CrowdStrike/falcon-helm/blob/main/helm-charts/falcon-kac/templates/_helpers.tpl#L189-L204